sdk/ai/Azure.AI.Agents.Persistent/samples/Sample13_PersistentAgents_OpenAPI.md

```diff
@@ -69,7 +69,7 @@ PersistentThreadMessage message = client.Messages.CreateMessage(
     MessageRole.User,
     "What's the weather in Seattle?");
 
-ThreadRun run = client.Runs.CreateRun(thread, agent);
+ThreadRun run = client.Runs.CreateRun(thread.Id, agent.Id);
```

Comment:
Why do we need these changes? It seems, we still have the signature client.Runs.CreateRun(thread, agent);